### DISCUSSION TITLE
update:checked-row-keys bug

### DISCUSSION THREAD

**Discussion by shi-sky** [tusen-ai/naive-ui#2617] [Q&A]
update:checked-row-keys  
我这边 删除表单数据
重新获取update:checked-row-keys  发现还会包含之前的 key

**Comment by whale2002** [comment]
你是怎么删除的，或者说....emmm 来个复现

**Reply by shi-sky** [reply]
let checkedData = [];
const cargoList = ref([]);
function onCheckedRow(rowKeys) {
  // 这里获取还是包含上次 key
  console.log('rowKeys', rowKeys)
  checkedData = rowKeys
}

function handDelList() {
 for (let i = 0; i < checkedData.length; i++) {
    const index = checkedData[i];
    const rowIndex = cargoList.value.findIndex((e) => e.index == index);
    if (rowIndex >= 0) cargoList.value.splice(rowIndex, 1);
  }
}

**Reply by 07akioni** [reply]
有 bug 的话请提个 issue，然后给出复现
